src/sources/hamonisedAttributes.json

```diff
@@ -0,0 +1,39 @@
+{
+    "physicalSpecimenId": {
+        "displayName": "Physical Specimen ID"
+    },
+    "ods:physicalSpecimenIdType": {
+        "displayName": "Physical Specimen ID Type"
+    },
+    "ods:organizationId": {
```

Comment:
Just a remark, this will be changed to organisation soon :smiling_face_with_tear:

Reply:
Yes, will update this in a new PR

Reply:
Included in the upcoming: Base Component Styling PR